JSQMessagesViewController/Factories/JSQMessagesToolbarButtonFactory.m

```diff
@@ -38,13 +55,14 @@ + (UIButton *)defaultAccessoryButtonItem
     accessoryButton.contentMode = UIViewContentModeScaleAspectFit;
     accessoryButton.backgroundColor = [UIColor clearColor];
     accessoryButton.tintColor = [UIColor lightGrayColor];
+    accessoryButton.titleLabel.font = self.buttonFont != nil ? self.buttonFont : [UIFont boldSystemFontOfSize:17.0];
```

Comment:
once we add the `init` above, we don't need this `nil` check and can just assign `self.buttonFont` 😄